src/components/CowPenContextMenu/CowPenContextMenu.js

```diff
@@ -220,104 +233,138 @@ export const CowCard = ({
   isNameEditable = !!handleCowNameInputChange,
 }) => {
   const [name, setName] = useState(cow.name)
+  const [cowImage, setCowImage] = useState(pixel)
+  const cardRef = useRef()
+  const scrollAnchorRef = useRef()
+
+  useEffect(() => {
+    ;(async () => {
+      setCowImage(await getCowImage(cow))
+    })()
+  }, [cow])
+
+  useEffect(() => {
+    if (isSelected) {
+      const { current: scrollAnchor } = scrollAnchorRef
+      const { current: card } = cardRef
+      if (!scrollAnchor || !card) return
+
+      // scrollIntoView is not defined in the unit test environment.
+      if (scrollAnchor.scrollIntoView && !isInViewport(card)) {
+        scrollAnchor.scrollIntoView({ behavior: 'smooth' })
+      }
+    }
+  }, [isSelected])
 
   return (
-    <Card {...{ className: 'cow-card', raised: isSelected }}>
-      {isSelected && (
-        <span className="visually_hidden">
-          {cow.name} is currently selected
-        </span>
-      )}
-      <CardHeader
+    <>
+      <a
         {...{
-          avatar: (
-            <img
-              {...{ src: animals.cow[cowColors[cow.color].toLowerCase()] }}
-              alt="Cow"
-            />
-          ),
-          title: (
-            <>
-              {isNameEditable ? (
-                <TextField
+          className: 'scroll-anchor',
+          href: `#cow-${cow.id}`,
+          ref: scrollAnchorRef,
+        }}
+      >
+        &nbsp;
+      </a>
+      <Card
+        {...{
+          className: classNames('cow-card', { 'is-selected': isSelected }),
+          raised: isSelected,
+          ref: cardRef,
+        }}
+      >
+        {isSelected && (
+          <span className="visually_hidden">
+            {cow.name} is currently selected
+          </span>
```

Comment:
This is actually an old change from https://github.com/jeremyckahn/farmhand/blame/86b92ad3a3df5cee3b71c1b633aa5d3ad85875ee/src/components/CowPenContextMenu/CowPenContextMenu.js#L227-L229. It's just showing up here because it was indented! 😄